client/python/README.md

```diff
@@ -0,0 +1,40 @@
+# Armada Python client
+
+
+Python client wrapping the gRPC services defined in `submit.proto` and `events.proto`; allows for
+
+- submitting, cancelling, and reprioritising jobs, and
+- watching for job events.
+
+
+
+## Build
+Prerequisites:
+
+For building the python client:
+In the repo level, run `make python`
+`cd client/python`
+`poetry install`
+
+1) pyenv
+    - Sets up local python environment for supporting multiple python environments
+    - Set up a local python 3.9 environment 
+2) poetry
+    - Package is defined by pyproject.toml
+    - `poetry install` will pull dependencies and install based on pyproject.toml
+3) tox
+    - `poetry run tox -e format` will format/lint your code according to default black settings and flake8
```

Comment:
nit: the format tox target only *checks* that the code matches black's expected formatting; we'd need to add a target without `--check` to allow folks to change it in place.